nextflow_schema.json

```
@@ -327,7 +327,7 @@
                 },
                 "cf_ploidy": {
                     "type": "string",
-                    "default": 2,
+                    "default": "2",
```

this is a weird update that happens with the schema builder. We should revert it because it has unintended consequences
```suggestion
                    "default": 2,
```

---

I included this change in my latest commit, but maybe it would have been cleaner to just commit your suggestion.